[KEP-5004] DRA Extended resource documentation (#51710)
kubernetes/website: content/en/docs/reference/command-line-tools-reference/feature-gates/DRAExtendedResource.md

---
title: DRAExtendedResource
content_type: feature_gate
_build:
  list: never
  render: false

stages:
  - stage: alpha
    defaultValue: false
    fromVersion: "1.34"
---

> Enables support for the [Extended Resource allocation by DRA](/docs/concepts/scheduling-eviction/dynamic-resource-allocation/#extended-resource) feature. It makes it possible to specify an extended resource name in a DeviceClass.

Reviewer: (nit)
I would link to a subsection of https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/ (which would then hyperlink to the DRA docs).

This framing is OK for alpha; at beta, we raise the quality expectation for feature docs.

Author: do you suggest to link to https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#extended-resources ? this section does not have DRA related information, not a good link to point to, IMO.

---
title: DRAExtendedResource
content_type: feature_gate
_build:
  list: never
  render: false

stages:
  - stage: alpha
    defaultValue: false
    fromVersion: "1.34"
---
Enables support for the [Extended Resource allocation by DRA](/docs/concepts/configuration/manage-resources-containers/#extended-resources-allocation-by-dra) feature.
It makes it possible to specify an extended resource name in a DeviceClass.

> This feature gate has no effect unless the `DynamicResourceAllocation` feature gate is enabled.

Reviewer: We should revise the `DynamicResourceAllocation` file so that people know to update this one when they lock `DynamicResourceAllocation` to enabled.

> This feature gate has no effect unless you also enable the `DynamicResourceAllocation` feature gate.

Reviewer: In v1.34 that gate is on by default, so we should reword. However you _can_ disable it (and your cluster then won't pass conformance…)

Author: hmmm, we used the same wording in other DRA alpha features, e.g. PrioritizedList (content/en/docs/reference/command-line-tools-reference/feature-gates/DRAPrioritizedList.md), do you suggest to change this in all other DRA alpha features? any suggestion on the wording?

> This feature gate has no effect unless you also enable the `DynamicResourceAllocation` feature gate.

Reviewer 2: @lmktfy maybe something like this?

```suggestion
This feature gate has no effect unless the `DynamicResourceAllocation` feature gate is enabled.
```
